<launch>
<arg name="frequency" value="40"/> 
<arg name="use_mmuav_simulation" value="false"/>
<arg name="config_name" value="mmuav_config.yml"/>

<group if="$(arg use_mmuav_simulation)">

    <include file="$(find mmuav_gazebo)/launch/uav_attitude_position.launch">
    		<arg name="manipulator_type" value="none" />
    		<arg name="manipulator_tool" value="none" />
    		<arg name="z" value="1.0"/>

  	</include>    

</group>

<node pkg="uav_ar_gui" type="firstui.py" name="uav_AR_gui_node" output="screen" args="$(arg frequency) $(arg config_name)"></node>


</launch>
